main.py

```diff
@@ -67,3 +68,34 @@ def power_on(settings: Annotated[Settings, Depends(get_settings)]):
         '<network-data xmlns="http://www.Bose.com/Schemas/2012-12/NetworkMonitor/" />'
         "</network-landscape></diagnostic-data></device-data>"
     )
+
+@app.get("/bmx/registry/v1/services",
+         tags=["bmx"]
+)
+def bmx_services(settings: Annotated[Settings, Depends(get_settings)]):
+    response = BmxResponse()
+    response._links = dict(bmx_services_availability = dict(href = '../servicesAvailability'))
+    response.askAgainAfter = 1277728
+    tunein = Service()
+    tunein._links = dict(bmx_navigate=Link(href="/v1/navigate"), bmx_token = Link(href="/v1/token"),
+        self = Link(href="/"))
+    tunein.askAdapter = False
+    tunein.assets = Asset(color="#000000",
+        description="With TuneIn on SoundTouch, listen to more than 100,000 stations and the hottest podcasts, "
+                    "plus live games, concerts and shows from around the world. However, you cannot access your "
+                    "Favorites and Premium content on your existing TuneIn account at this time.",
+        icons=IconSet(defaultAlbumArt="https://media.bose.io/bmx-icons/tunein/default-album-art.png",
+                      largeSvg="https://media.bose.io/bmx-icons/tunein/smallSvg.svg",
+                      monochromePng="https://media.bose.io/bmx-icons/tunein/monochromePng.png",
+                      monochromeSvg="https://media.bose.io/bmx-icons/tunein/monochromeSvg.svg",
+                      smallSvg="https://media.bose.io/bmx-icons/tunein/smallSvg.svg"),
```

Comment:
could you make either a ticket or a comment so we don't forget these two?

1. these should all be read from a set of constants (or a config file, if people might want to change things.
1. also if the icons render, we should download local copies of them as assets, if licensing allows, or make homemade replacements if not.